Mathlib/Analysis/Analytic/Constructions.lean

```diff
@@ -102,15 +102,15 @@ theorem AnalyticWithinAt.add (hf : AnalyticWithinAt 𝕜 f s x) (hg : AnalyticWi
   (hpf.add hqf).analyticWithinAt
 
 @[fun_prop]
-theorem AnalyticAt.add (hf : AnalyticAt 𝕜 f x) (hg : AnalyticAt 𝕜 g x) : AnalyticAt 𝕜 (f + g) x :=
+theorem AnalyticAt.fun_add (hf : AnalyticAt 𝕜 f x) (hg : AnalyticAt 𝕜 g x) :
+    AnalyticAt 𝕜 (fun z ↦ f z + g z) x :=
   let ⟨_, hpf⟩ := hf
   let ⟨_, hqf⟩ := hg
   (hpf.add hqf).analyticAt
 
 @[fun_prop]
-theorem AnalyticAt.add' (hf : AnalyticAt 𝕜 f x) (hg : AnalyticAt 𝕜 g x) :
```

Comment:
The lemma `AnalyticAt.add'` does not exist any more after the PR, so you should deprecate it, in the form
```
@[deprecated (since := "2025-03-11") alias AnalyticAt.add' := AnalyticAt.fun_add
```
Same thing for the other lemmas that disappear in this PR.

Reply:
Done. Thanks!